jekyll/_cci2/authentication/mfa.adoc

```diff
@@ -0,0 +1,69 @@
+. You may close the window to complete the setup. When MFA is successfully configured, it is marked as **Enabled** on the Password & authentication page.
+
+image::../../img/docs/authentication/mfa-enabled.png[Password and authentication page with MFA enabled]
+
+[#using-mfa]
+== Using MFA on your account
+
+Once you have MFA enabled, then you must provide a second verification factor whenever you log into your account.
+
+image::../../img/docs/authentication/mfa-otp-login.png[Login page with request for OTP]
+
+You will also be asked to provide a second factor whenever you attempt to change the authentication settings of your account (for example updating your email or password).
+
+Currently CircleCI supports MFA using authenticator applications only. Once you have configured MFA, as per the previous section, then you must use the OTP generated by
+the application whenever this is requested in the CircleCI web app. The OTP will be displayed in the authenticator app, and you must input this code when prompted in the CircleCI web app.
+
+[#mfa-recovery-codes]
+== MFA recovery codes
+
+WARNING: If you have MFA enabled, and lose access to your MFA factor and recovery code, then you will lose access to your account. You must ensure that you do not lose the recovery code.
+
+If you lose access to your MFA factor (for example, by losing access to the authenticator application), you may use the recovery code as a second factor instead. A recovery code may only be used once, after which a new recovery code is generated and shared with you in the CircleCI web app. Whenever a new recovery code is generated, it is important to always save it somewhere safe.
+
+It is also possible to intentionally regenerate your MFA recovery code. To do this, follow these steps:
+
+. In the link:https://app.circleci.com/home/[CircleCI web app], select your profile from the upper right corner, then select **User Settings**.
+. Select **Password & authentication** from the sidebar.
+. On the Password & authentication page, in the **Multi-factor authentication**section, select **Add/edit authenticator app**.
+. Input your password at the password prompt.
+. Input your OTP at the OTP prompt. Once these have been successfully submitted, your OTP ID and recovery code ID will be displayed.
+. Select the button to **Regenerate recovery code** on the recovery code row.
+. At the prompt to regenerate the recovery code, select `Yes, regenerate code`. This will generate a new code and invalidate the previous recovery code. Store the new code somewhere safe. It will not be displayed again.
+
+image::../../img/docs/authentication/mfa-regenerate-recovery-code.png[Password and authentication page with regenerate recovery code button]
+
+image::../../img/docs/authentication/mfa-display-recovery-code.png[Password and authentication page displaying new recovery code]
```

Comment:
@emilywoods I'm wondering if we can come up with a few troubleshooting/FAQs for this page?

Reply:
Yes, for sure! Would it be ok for me to add this in a follow-up pr?